js-api-doc/README.md

```diff
@@ -0,0 +1,60 @@
+Both [Dart Sass] and [LibSass] support the same JavaScript API. Dart Sass is
+distributed as the pure-Javascript [`sass` package], and LibSass is distributed
+as a native extension in the [`node-sass` package].
+
+[Dart Sass]: https://sass-lang.com/dart-sass
+[LibSass]: https://sass-lang.com/libsass
+[`sass` package]: https://www.npmjs.com/package/sass
+[`node-sass` package]: https://www.npmjs.com/package/node-sass
+
+## Usage
+
+The JavaScript API has two entrypoints for compiling a Sass to CSS. Each one can
+compile either a Sass file by passing in [[LegacyFileOptions]] or a string of
+Sass code by passing in a [[LegacyStringOptions]].
+
+* [[renderSync]] runs synchronously. It's **by far the fastest option** when
+  using Dart Sass, but at the cost of only supporting synchronous {@link
+  LegacyImporter | importer} and {@link LegacyFunction | function} plugins.
+  
+  ```js
+  var sass = require('sass'); // or require('node-sass');
```

Comment:
Nit (here and throughout): Probably makes sense to use `const`/`let` instead of `var`.